packages/typespec-ts/test/integration/generated/azure/resource-manager/common-properties/src/index.d.ts

```diff
@@ -242,7 +241,7 @@ export declare interface ResourceModelWithAllowedPropertySet extends TrackedReso
 export declare interface ResourceModelWithAllowedPropertySetOutput extends TrackedResourceOutput {
     managedBy?: string;
     kind?: string;
-    eTag?: string;
+    readonly eTag?: string;
```

Comment:
refer to : https://github.com/Azure/typespec-azure/pull/1881/files#diff-823f10c1812ffa803f58da35c7d41623a79c2e5c37f0c15d09ce66bcfdcfbe97